src/utils/numbers.ts

```diff
@@ -47,9 +47,6 @@ export function toString(x: number): string {
   return x.toString()
 }
 
-/** @deprecated use parseFraction instead
- * @example parseFraction(value)
- */
```

Comment:
parseFraction does not divide decimals

Reply:
divide decimals by yourself after `parseFraction`

Reply:
I prefer this function
```
export const uint256ToFraction = (value: string, decimals = RESERVE_USD_DECIMALS) =>
  new Fraction(value, JSBI.exponentiate(JSBI.BigInt(10), JSBI.BigInt(decimals)))
  ```
  `uint256ToFraction(value)` is simplify.
  I dont want to use it likes: `parseFraction(value).divide(BigInt(decimals))` in every where
  
  I don't know any bug in your parseFraction function :trollface: